### DISCUSSION TITLE
[SQL Editor][Database Navigator]: table declaration is always opening in default schema

### DISCUSSION THREAD

**Discussion by PetbkA** [dbeaver/dbeaver#12496] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 10 Ent x64 LTSC (version 1809)
- DBeaver version: 21.0.3.202104181339
- Additional extensions: None

#### Connection specification:
- Database name and version: PostgreSQL 12.5 x64
- Driver name: PostgreSQL JDBC Driver 42.2.18.jre7
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No 

#### Describe the problem you're observing:
Table Declaration is always opening in the default schema, not in the one you've selected in Database Navigator or SQL Editor. Reconnecting after changing the schema in SQL Editor doesn't help.

The same problem was [raised](https://dbeaver.io/forum/viewtopic.php?f=2&t=1313) a long time ago on the forum and should be fixed, but it still persists in the latest version.

#### Steps to reproduce, if exist:
1. Set a default schema in DB connection via _Driver Properites → currentSchema_;
2. Open SQL Editor and set the _non-default_ schema in _Active catalog/schema_;
3. Open the declaration of some table via F4 or _Open Declaraion_ in right mouse button menu;
4. Repeat the same by selecting table in non-default schema via Database Navigator

**Comment by uslss** [comment, marked as answer]
Probably, it should help if you disable those options:
![image](https://user-images.githubusercontent.com/31996417/116541093-af823f00-a8f3-11eb-926f-17474e4e3265.png)
![image](https://user-images.githubusercontent.com/31996417/116541156-c9238680-a8f3-11eb-9284-36b2901521aa.png)

**Comment by PetbkA** [comment]
> Probably, it should help if you disable those options:
> ![image](https://user-images.githubusercontent.com/31996417/116541093-af823f00-a8f3-11eb-926f-17474e4e3265.png)

Thank you! Changing this one and restarting DBeaver helped. Although the behavior is still strange and not expected
